core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -1687,6 +1688,38 @@ object FundingSignaturesV0TLV extends TLVFactory[FundingSignaturesV0TLV] {
 
 sealed trait DLCSetupTLV extends TLV
 
+case class DLCOfferMessageTLV(
+    peer: NormalizedString,
+    message: NormalizedString,
+    offer: DLCOfferTLV)
+    extends DLCSetupTLV {
+
+  require(peer.length <= 1024, "peer length must not exceed 1024 characters")
+  require(message.length <= 1024,
+          "message length must not exceed 1024 characters")
+
+  override val tpe: BigSizeUInt = DLCOfferMessageTLV.tpe
+
+  override val value: ByteVector = {
+    strBytes(peer) ++ strBytes(message) ++ offer.bytes
+  }
+}
+
+object DLCOfferMessageTLV extends TLVFactory[DLCOfferMessageTLV] {
+
+  override val tpe: BigSizeUInt = BigSizeUInt(65534)
```

Comment:
@nkohen is there a good way to pick TLV  numbers?

Reply:
I usually use the next even number

Reply:
next relative to what?